core/modules/admin_bar/js/admin_bar.js

```diff
@@ -412,11 +412,21 @@ Backdrop.adminBar.behaviors.search = function (context, settings, $adminBar) {
       e.preventDefault();
     }
     if (show) {
-      $adminBar.find('.active-search-item').removeClass('active-search-item');
+      if ($adminBar.find('.active-search-item').classList.length === 1) {
+        $adminBar.find('.active-search-item').removeAttr('class');
+      }
+      else {
+        $adminBar.find('.active-search-item').removeClass('active-search-item');
+      }
       $(this).addClass('active-search-item');
```

Comment:
Where does this "classList" in line 415 come from?

Another approach: after `removeClass` check for an empty attribute and remove then:

`$('*[class=""]').removeAttr('class');`

Also untested... ;)

Reply:
Sorry for the late response @indigoxela 😓 

Here's some info on `classList`:

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList
https://www.w3schools.com/jsref/prop_element_classlist.asp

Reply:
...this basically searches for the element with a class of `.active-search-item`, and if that is the only class, it removes the entire `class=` attribute instead of only the class.

Reply:
Ah, got it, thanks for the hint.